tasks/system_probe.py

```diff
@@ -85,10 +82,9 @@ def build(
                 maj_ver=maj_ver, min_ver=min_ver, patch_ver=patch_ver, target_arch=windres_target
             )
         )
-    else:
-        if compile_ebpf:
-            # Only build ebpf files on unix
-            build_object_files(ctx, bundle_ebpf=bundle_ebpf)
```

Comment:
Could we remove `bundle_ebpf` from here https://github.com/DataDog/datadog-agent/blob/a5fac57539a6e4a0f5695d64a0bf12d0864fd444/tasks/system_probe.py#L51
and remove the bundle tag from the builds tags here? https://github.com/DataDog/datadog-agent/blob/a5fac57539a6e4a0f5695d64a0bf12d0864fd444/tasks/system_probe.py#L95-L96

Reply:
We have to leave bundling in for the runtime-security team.

Reply:
> always add the bundle tag to the build tags here?

We don't want to include the bundle tag for system-probe builds unless requested (which is only the runtime-security folks now).